test/tests_split_extremum_y.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+# coding:utf-8
+"""
+  SoftKnit21 framework
+
+License
+=======
+
+ Copyright (C) <2019>  <Odile Troulet-Lamberttex@orange.fr>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as
+    published by the Free Software Foundation, either version 3 of the
+    License, or  any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+
+Module purpose
+==============
+
+Tests for pathPoints
+
+Implements
+==========
+
+
+
+Documentation
+=============
+
+Usage
+=====
+
+
+@author: Odile Troulet-Lambert
+@copyright: Odile Troulet-Lambert
+@license: GPL

```

Comment:
GPL is significantly more restrictive than the MIT license the project is under.